### PR TITLE
updating caption in kevin's md file

### LINKED ISSUE #3: Add A New Slide
## We Need More Slides
Grab another octocat and create a new slide from scratch.
If you need help, check out the Appendix in the training manual.

### PR DESCRIPTION
**Pull request by pw-kyu**
# sample header

- [x] sample task 1
- [x] sample task 2

resolves #3

### CODE REVIEW

File: _slides/23-pw-kyu.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+kyu caption
```
Comment (ArthurPricePW) [inline _slides/23-pw-kyu.md:6]: nice

**Review by erhernandezPW — COMMENTED** [review]
ez pass

**Review by erhernandezPW — APPROVED** [review]
pass

File: _slides/23-pw-kyu.md (hunk 1 above)
Comment (pw-kyu) [reply _slides/23-pw-kyu.md:6]: i apologize king arthur

**Pull request merged**